src/utils_db_query.c

```diff
@@ -78,6 +80,7 @@ typedef struct udb_result_preparation_area_s udb_result_preparation_area_t;
 struct udb_query_preparation_area_s /* {{{ */
 {
   size_t column_num;
+  size_t plugin_instance_pos;
```

Comment:
Option `PluginInstanceFrom` set on `Query` level, not inside `Result`, so  there is no need to calculate field position for each result.